sql/atxs/atxs_test.go

```diff
@@ -1201,3 +1201,56 @@ func Test_AtxWithPrevious(t *testing.T) {
 		require.Equal(t, atx2.ID(), id)
 	})
 }
+
+func Test_FindDoublePublish(t *testing.T) {
```

Comment:
I think this is missing a test case for two atxs containing a post by the same ID but signed by different IDs:

```go
t.Run("double publish different smesher", func(t *testing.T) {
	t.Parallel()
	db := sql.InMemory()

	sig1, err := signing.NewEdSigner()
	require.NoError(t, err)

	atx0, blob := newAtx(t, sig1)
	require.NoError(t, atxs.Add(db, atx0, blob))
	require.NoError(t, atxs.SetUnits(db, atx0.ID(), atx0.SmesherID, 10))
	require.NoError(t, atxs.SetUnits(db, atx0.ID(), sig.NodeID(), 10))

	sig2, err := signing.NewEdSigner()
	require.NoError(t, err)

	atx1, blob := newAtx(t, sig2)
	require.NoError(t, atxs.Add(db, atx1, blob))
	require.NoError(t, atxs.SetUnits(db, atx1.ID(), atx1.SmesherID, 10))
	require.NoError(t, atxs.SetUnits(db, atx1.ID(), sig.NodeID(), 10))

	atxIDs, err := atxs.FindDoublePublish(db, sig.NodeID(), atx0.PublishEpoch)
	require.NoError(t, err)
	require.ElementsMatch(t, []types.ATXID{atx0.ID(), atx1.ID()}, atxIDs)
})
```